SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -319,7 +320,10 @@ const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_NON_AUTH_CLIENT_INTERES
 // A list of components clients require on entities they are authoritative over on top of the components already checked out by the interest
 // query.
 const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_AUTH_CLIENT_INTEREST =
-	TArray<Worker_ComponentId>{ // RPCs from the server
+	TArray<Worker_ComponentId>{ // Actor components
+								VISIBLE_COMPONENT_ID,
```

Comment:
Should this be in the auth client interest?
I thought it would have to be in the non-auth client interest?